### ISSUE TITLE
r2l isotope volr fields should probably be added to r2x_fluxes, not r2x_states

### ISSUE THREAD

**Issue by billsacks** [ESMCI/cime#2791]
@swensosc noticed this issue: currently, the water isotope volr fields are added to r2x_states:

https://github.com/ESMCI/cime/blob/442c6a60f763b197645fbd7a2130f851b42daeca/src/drivers/mct/shr/seq_flds_mod.F90#L2998-L3016

The main volr field is in r2x_fluxes, so we're thinking that the isotope ones should be there, too. Currently, r2x_states doesn't seem to be mapped at all, but in some work @swensosc is doing, he's going to start mapping r2x_states using a state (bilinear) mapper.

@cacraigucar I think you're the one who added this code. Do you have any thoughts on this?

**Comment by cacraigucar** [comment]
Unfortunately, I was an integrator with the isotope code  and not the developer.  Those changes were in code I inherited when I took over the project.  The developers whose names are attached to the original changes in the svn repo are @ekluzek and jesse.nusbaumer@nasa.gov.

**Comment by ekluzek** [comment]
I agree that the isotope fields should mirror the bulk fields. Had I noticed this, I would've pointed it out as a problem as well. It could also be in the history, if it was consistent in the beginning and then the bulk was changed, but the isotopes weren't. We could search the history and look into that. But, it's just pretty clear that isotopes should be handled the same as bulk, so I don't know that we need to look into the history.
